### PR TITLE
feat(global-banner): v2 updates

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
[#1610](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/issues/1610)

### Description
This PR updates any mentions of the Universal banner name to Global banner, and changes the Resources to appear right before the Content guidance section.

### Changelog

**Changed**

- changed Universal banner to Global banner
- changed the location of Resources to right before the content guidance section

### CODE REVIEW

File: src/pages/components/global-banner.mdx [hunk 1]
```diff
@@ -22,21 +22,19 @@ import ResourceLinks from 'components/ResourceLinks';
...
 </AnchorLinks>
 
-<ResourceLinks name="Universal banner" type="layout" />
-
 ## Overview
 
-Universal banner is intended for high level and critical announcements such as IBM's THINK global event or COVID-19 messaging. The banner is the only component that can be positioned above the masthead. It has a max height to encourage succinct messaging and optional fields for an image, heading, body copy, and CTA.
+Global banner is intended for high level and critical announcements such as IBM's THINK global event or COVID-19 messaging. The banner is the only component that can be positioned above the masthead. It has a max height to encourage succinct messaging and optional fields for an image, heading, body copy, and CTA.
 
 For non-critical announcements and callouts, use the [Content group banner](https://www.ibm.com/standards/carbon/components/content-group-banner/) component instead.
```
Comment (ariellalgilmore) [inline src/pages/components/global-banner.mdx:29]: I believe content-group-banner got renamed to In page banner

**Review by ariellalgilmore — COMMENTED** [review]
LGTM! just one small comment and when looking at the component overview/gallery pages `global-banner` still appears at the end. I think they are in alphabetical order, so might need to rearrange the positions in the data folder?

**Review by ariellalgilmore — APPROVED** [review]
lgtm!!

**Review by oliviaflory — APPROVED** [review]
Thanks @IgnacioBecerra !!!

**Pull request merged**